### ISSUE TITLE
[BUG] MIDI note turns off automatically after 10 minutes. [midi-relay]

### ISSUE THREAD

**Issue by kwonsangjun** [bitfocus/companion#1710]
I am controlling the stage lighting using a midi-relay module in the streamdeck and companion. But when I turn on the light by sending a MIDI signal, the MIDI note turn off automatically after exactly 10 minutes. The same symptoms appear in the MIDI of the StreamDeck basic app. Which part has the function to turn off after 10 minutes?

RTP midi interface : iconnectivity mioXL

**Comment by kraves-oz** [comment]
This sounds like it is the end device turning off after 10minutes. Perhaps you need a long delay on the button that then fired itself again.  As a form of keep awake.

**Comment by kwonsangjun** [comment]
> This sounds like it is the end device turning off after 10minutes. Perhaps you need a long delay on the button that then fired itself again. As a form of keep awake.

yes... so we try midi note on again quickly.  I dont know where cut Note on message.
